loom/common/core/stringTable.c

```diff
@@ -61,15 +61,15 @@ static unsigned long hash(const char *str)
 
 static stringTableEntry_t *allocEntry(const char *str)
 {
+    size_t len;
     stringTableEntry_t *entry = (stringTableEntry_t *)lmAlloc(NULL, sizeof(stringTableEntry_t));
 
     entry->next = NULL;
+    len = strlen(str);
+    entry->string = (const char*)lmAlloc(NULL, len + 1);
+    memcpy((char*)entry->string, str, len);
+    ((char*)entry->string)[len] = '\0';
 
-#if LOOM_COMPILER != LOOM_COMPILER_MSVC
-    entry->string = (const char *)strdup(str);
```

Comment:
I think we should just use utString, but this seemed like a class isolated enough that I just added a manual strdup.